app/screens/more_channels/more_channels.js

```diff
@@ -27,24 +28,27 @@ import {
     setNavigatorStyles,
     getKeyboardAppearanceFromTheme,
 } from 'app/utils/theme';
+import BottomSheet from '../../utils/bottom_sheet';
```

Comment:
```suggestion
import BottomSheet from 'app/utils/bottom_sheet';
```

and try to sort the imports